src/main/java/com/microsoft/spring/data/gremlin/annotation/EdgeSet.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.annotation;
+
+import org.springframework.data.annotation.Persistent;
+
+import java.lang.annotation.*;
+
+/**
+ * Specifies the field as EdgeSet of graph.
```

Comment:
put those above? and consistent in all files

Reply:
seems mongodb and neo4j use this style here. Do we need to align with them?

Reply:
no need. actually we don't add @author that in codes, except license disclaimer. but add <developer> in pom.xml, you could check azure-spring-boot